app/api/schema/event_invoices.py

```diff
@@ -46,7 +46,7 @@ class Meta:
     last4 = fields.Str(allow_none=True)
     stripe_token = fields.Str(allow_none=True)
     paypal_token = fields.Str(allow_none=True)
-    status = fields.Str(validate=validate.OneOf(choices=["paid", "due"]), allow_none=True)
+    status = fields.Str(validate=validate.OneOf(choices=["paid", "due", "refunding", "refunded", "unpaid", "failed"]), allow_none=True)
```

Comment:
Unpaid and due are same or different @mariobehling

Reply:
Also, `overdue` is derivable for `due + current_date > due_date` so we can find which invoices are overdue.